lua/quickmath/init.lua

```diff
@@ -24,7 +24,7 @@ local function StartSession()
 
 		for i, line in ipairs(content) do
 			local line = content[i]
-			if not string.find(line, "^%w+%s*=") and not string.find(line, "^%s*$") then
+			if not string.find(line, "^[A-Za-z0-9_]+%s*=") and not string.find(line, "^%s*$") then
```

Comment:
Wouldn't this match "1234"?

Reply:
Oh yeah, I forgot variables can't start with a digit, thanks for reminding me